doc/api/stream.md

```diff
@@ -61,6 +61,14 @@ Stream instances are switched into object mode using the `objectMode` option
 when the stream is created. Attempting to switch an existing stream into
 object mode is not safe.
 
+The object mode indicates that the stream either accepts and / or emits objects.
+Be aware that streams which operate in the object mode can only pipe to streams
+which are also in the object mode. Otherwise an error will be thrown. To pipe a
```

Comment:
I don’t think this is accurate – you can do this when the objects that are passed are strings and/or Buffers.

Reply:
This was so long ago, I am not quite certain, but I was quite sure to have found some undocumented behaviour. Either way I rewrote the paragraph to reflect your response.